src/main/java/com/epam/page/object/generator/adapter/annotation/IJavaAnnotation.java

```diff
@@ -0,0 +1,10 @@
+package com.epam.page.object.generator.adapter.annotation;
+
+import java.util.List;
+
+public interface IJavaAnnotation {
```

Comment:
Please do not use "I" prefix to mark the interface

Reply:
Done